public/js/components/facets-view/facets-view.js

```
@@ -263,6 +282,57 @@ function FacetsViewController($http, $scope) {
+      // Clear all child nodes
+    groupNode.childNodes.length = 0;
+
+    var hasCheckedArtifactFacets = false;
+
+    for (var setting of ctrl.viewModel[DatabusUris.DATABUS_ARTIFACT_PROPERTY].visibleFacetSettings) {
+      hasCheckedArtifactFacets = hasCheckedArtifactFacets || setting.checked;
+    }
+
+    if (hasCheckedArtifactFacets) {
+
+      for (var setting of ctrl.viewModel[DatabusUris.DATABUS_ARTIFACT_PROPERTY].visibleFacetSettings) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'setting'. Consider to delete it or rename it.

'setting' is defined here:


(lint/suspicious/noRedeclare)